scripts/backend_tests_incomplete_coverage.txt

```diff
@@ -90,6 +90,8 @@ core.utils_test
 main_test
 scripts.build_test
 scripts.concurrent_task_utils_test
+# TODO(#19499): Fix topological sort coverage.
+scripts.create_topological_sort_of_all_services_test
```

Comment:
@nikitaevg I don't understand, why did this become uncovered? It seems tangential to this PR.

Reply:
This script parses TypeScript imports. Imports are getting formatted, hence the old parsing stops working.

I'll either fix the script in the next PR or will delete it, I don't see any references to it neither from wiki nor from the code. Maybe you know something about it?

Reply:
No idea what this is for either. Let's delete it, thanks!